src/client/src/Client.c

```diff
@@ -146,6 +232,19 @@ STATUS createKinesisVideoClient(PDeviceInfo pDeviceInfo, PClientCallbacks pClien
 
     pKinesisVideoClient->base.pStateMachine = pStateMachine;
 
+    if(pKinesisVideoClient->deviceInfo.clientInfo.automaticStreamingFlags == AUTOMATIC_STREAMING_INTERMITTENT_PRODUCER) {
+        if ( !IS_VALID_TIMER_QUEUE_HANDLE(pKinesisVideoClient->timerQueueHandle) ) {
+            // Create timer queue
+            CHK_STATUS(timerQueueCreate(&pKinesisVideoClient->timerQueueHandle));
+            // Store callback in client so we can override in tests
+            pKinesisVideoClient->timerCallbackFunc = checkIntermittentProducerCallback;
+            CHK_STATUS(timerQueueAddTimer(pKinesisVideoClient->timerQueueHandle, 200 * HUNDREDS_OF_NANOS_IN_A_MILLISECOND,

```

Comment:
Define the start of the timer in an internal header and select something in a few seconds instead of millis. The application might still be doing a lot of startup/construction when this starts firing the first time which is undesirable